utils/errors/notifier/notifier.go

```diff
@@ -242,28 +243,22 @@ func doNotify(err error, skip int, level string, rawData ...interface{}) error {
 			fields = append(fields, &rollbar.Field{Name: "traceId", Data: traceID})
 		}
 		fields = append(fields, &rollbar.Field{Name: "server", Data: map[string]interface{}{"hostname": getHostname(), "root": getServerRoot()}})
-		rollbar.ErrorWithStack(level, errWithStack, convToRollbar(errWithStack.StackFrame()), fields...)
+		rollbar.ErrorWithStack(sev.String(), errWithStack, convToRollbar(errWithStack.StackFrame()), fields...)
 	}
 
 	if sentryInited {
-		defLevel := raven.ERROR
-		if level == "critical" {
-			defLevel = raven.FATAL
-		} else if level == "warning" {
-			defLevel = raven.WARNING
-		}
 		ravenExp := raven.NewException(errWithStack, convToSentry(errWithStack))
 		packet := raven.NewPacketWithExtra(errWithStack.Error(), parsedData, ravenExp)
 
 		for _, tags := range tagData {
 			packet.AddTags(tags)
 		}
 
-		packet.Level = defLevel
+		packet.Level = sev.RavenSeverity()
```

Comment:
Instead of doing `sev.RavenSeverity()`, IMO, it would be better to expose a func that convert `sev` object into `raven.Severity` in this package. I don't think the `.RavenSeverity` is in use other than here (ie: when we want to send it to Sentry)

Reply:
maintaining the wrapper for converting types as util func instead of as a struct function sounds good to me.

Reply:
In the end, I just removed the struct function and do type conversion directly since it's just a single use case. 🙏